packages/graphql-language-service-types/src/index.ts

```diff
@@ -283,11 +293,24 @@ export type CustomValidationRule = (
 
 export type Diagnostic = DiagnosticType;
 
+export type CompletionItemBase = {
+  label: string;
+  isDeprecated?: boolean;
+};
+
 export type CompletionItem = CompletionItemType & {
   isDeprecated?: boolean;
   deprecationReason?: string;
 };
 
+export type CompletionItemForCodeMirror = {
+  label: string;
```

Comment:
following above, let's combine completion items.

Reply:
Hmm. apparently, the completion items for vscode/monaco is kind of different than the  completion items for codemirror. I thought if we combined these items, it would make it difficult to see which items are for vscode/monaco, for codemirror, or for both. Should we combine them?

Reply:
Or, instead of exporting these type aliases and functions I'm trying to export in other [PR](#1264), how about having these in `codemirror`. (Added functions is the same as the ones in `graphql-language-service-interface` but accept different type.)